### PR TITLE
path-item-action: minor visual tweaks

### PR DESCRIPTION
**Pull request by jakob223**
Issue: KBFS-4023

### CODE REVIEW

File: shared/fs/common/path-item-action/menu.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ const hideMenuOnClick = (onClick: (evt?: SyntheticEvent<>) => void, hideMenu: ()
 }
 
 const makeMenuItems = (props: Props, hideMenu: () => void) => [
+  'Divider',
```
Comment (songgao) [inline shared/fs/common/path-item-action/menu.js]: Somehow this didn't show up for me on mobile. I guess it's just not supported on mobile. Perhaps this divider should be part of the header?

File: shared/fs/common/path-item-action/header.js [hunk 2]
```diff
@@ -34,8 +33,7 @@ class Header extends React.PureComponent<Props> {
   render() {
     return (
       <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true} style={styles.container}>
-        <PathItemIcon path={this.props.path} size={32} style={styles.pathItemIcon} />
-        <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />
+        <PathItemIcon path={this.props.path} size={48} style={styles.pathItemIcon} />
```
[the remark this answers is not in the document]
Comment (jakob223) [reply shared/fs/common/path-item-action/header.js]: for some reason it looks like `PathItemIcon`s only take `size` and it's a number.

File: shared/fs/common/path-item-action/menu.js (hunk 1 above)
Comment (jakob223) [reply shared/fs/common/path-item-action/menu.js]: It looks like we intentionally strip out dividers on mobile for some reason: https://github.com/keybase/client/blob/b4cb5dc540dac5126a783a8afeaf0374073c3d07/shared/common-adapters/floating-menu/menu-layout/index.native.js#L53-L58

File: shared/fs/common/path-item-action/header.js [hunk 3]
```diff
@@ -34,8 +33,7 @@ class Header extends React.PureComponent<Props> {
   render() {
     return (
       <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true} style={styles.container}>
-        <PathItemIcon path={this.props.path} size={32} style={styles.pathItemIcon} />
-        <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />
```
[the remark this answers is not in the document]
Comment (jakob223) [reply shared/fs/common/path-item-action/header.js]: Reversed on mobile for now.

File: shared/fs/common/path-item-action/menu.js (hunk 1 above)
Comment (jakob223) [reply shared/fs/common/path-item-action/menu.js]: Regardless, it feels like there is a gap on mobile (er, on Android anyway) by virtue of the way the menus work over there. @cecileboucheron was the gap in KBFS-4023 supposed to be added on both mobile and desktop, or just desktop?

Comment (cecileboucheron) [reply shared/fs/common/path-item-action/menu.js]: Weird... There should be a divider on both desktop and mobile between the info and the actions.

Comment (jakob223) [reply shared/fs/common/path-item-action/menu.js]: Okay, I'm changing the code to allow dividers as the first element on mobile in a FloatingMenu.

**Comment by jakob223** [comment]
Bump @songgao ? or are you waiting for mobile tooltip to be merged?

File: shared/fs/common/path-item-action/header.js [hunk 4]
```diff
@@ -34,9 +34,9 @@ class Header extends React.PureComponent<Props> {
...
+        <PathItemIcon path={this.props.path} size={48} style={styles.pathItemIcon} />
         <Kb.Box2 direction="horizontal" style={styles.nameTextBox}>
+          {Styles.isMobile && <StaticBreadcrumb pathElements={Types.getPathElements(this.props.path)} />}
```
Comment (songgao) [inline shared/fs/common/path-item-action/header.js]: Did you mean to move this inside this box?

Comment (jakob223) [reply shared/fs/common/path-item-action/header.js]: No.. fixing.

**Pull request merged**